GoogleMeetup/src/main/webapp/script.js

```diff
@@ -12,6 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The relevant meetup information
+var attendees = [];
+var destinationType = null;
```

Comment:
Might be better to attach these to something rather than keep them as global variables.